nonebot/adapters/onebot/v11/event.py

```diff
@@ -485,7 +485,7 @@ class FriendRequestEvent(RequestEvent):
 
     request_type: Literal["friend"]
     user_id: int
-    comment: str
+    comment: Optional[str]
```

Comment:
这里还需要加上 None，不然 Pydantic v2 会报错。

```suggestion
    comment: Optional[str] = None
```